.gitignore

```diff
@@ -4,8 +4,10 @@ cmake-build-*/
 
 # Visual Studio 2022
 .vs/
-build/
-lib/
+/build/
+/bin/
+/lib/
```

Comment:
these two aren't needed anymore, if you want to remove them while you're at it I'd be grateful